crates/driver/src/tests/setup/solver.rs

```diff
@@ -197,20 +197,24 @@ impl Solver {
                                 })
                             },
                         ));
-                        prices_json.insert(
-                            config
-                                .blockchain
-                                .get_token_wrapped(fulfillment.quoted_order.order.sell_token),
-                            fulfillment.execution.buy.to_string(),
-                        );
-                        prices_json.insert(
-                            config
-                                .blockchain
-                                .get_token_wrapped(fulfillment.quoted_order.order.buy_token),
-                            (fulfillment.execution.sell
-                                - fulfillment.quoted_order.order.surplus_fee())
-                            .to_string(),
-                        );
+                        assert!(prices_json
```

Comment:
nit: Having such big expressions is hard to read and less expressive than properly named local variables.
```rust
                        let previous_value = prices_json
                            .insert(
                                config
                                    .blockchain
                                    .get_token_wrapped(fulfillment.quoted_order.order.sell_token),
                                fulfillment.execution.buy.to_string(),
                            );
                        assert_eq!(previous_value, None, "existing price overwritten");
```